### ISSUE TITLE
Scope name problem when conventing from LiuZhuang13's code

### ISSUE THREAD

**Issue by leepp15** [shicai/DenseNet-Caffe#22]
Hello, Shicai. When generating train_val.prototxt from LiuZhuang13's code, the scope name will be "Convolution1", "BatchNorm13" etc., which is different from yours like "conv1/bn1", "conv3_2/x2/scale". I wonder that the difference will lead a failure to load the pre-train caffemodel. How to deal with the problem? And I will be very appreciated if you provide your make_densenet.py code. Thanks.

**Comment by shicai** [comment]
they are automatically named, while the layer names of prototxt files in my repo are manually set. 
you can download both weights files and prototxt files from my repo.

**Comment by leepp15** [comment]
But prototxt from your repo is deploy.prototxt. Revision work for initialization in every layer is quite miscellaneous. Could you please release the train_val.prototxt? Thanks a lot. :)

**Comment by shicai** [comment]
i did not have train_val.prototxt, you can write scripts to generate it.
